src/commands/deployment/get.test.ts

```diff
@@ -4,12 +4,12 @@ import {
   enableVerboseLogging,
   logger
 } from "../../logger";
-import { Helper, OUTPUT_FORMAT } from "./helper";
+import * as Get from "./get";
```

Comment:
Having to do `import * as <something> from ...` because `jest.spyOn` only takes in a name of a module, and not an exported const member